changelog.d/20251217_150504_athornton_DM_53638_simplectrs.md

```diff
@@ -0,0 +1,15 @@
+<!-- Delete the sections that don't apply -->
+
+### Backwards-incompatible changes
+
+- `config.lab.tmpSource` renamed to `config.lab.emptyDirSource`.
+
+### New features
+
+- A standard homedir provisioner can be run by setting `config.lab.standardInithome` to `true`. It requires that the controller be able to write as an administrative user to the volume containing user home directories.
+
+- `config.lab.homeVolumeName` may be set to tell the controller which volume contains user home directories.
+
+### Other changes
+
+- A standard `startup` initContainer will always be launched in a Lab Pod.
```

Comment:
Might be worth adding a few more lines here (Unless you want to keep things concise). Im thinking:

  - A new `/lab_startup` emptyDir volume is always created to facilitate communication between init containers and the lab container.                                                                               
  - On sites where `config.lab.env.RSP_SITE_TYPE` is set to `"science"`  an additional `landingpage` initContainer will run to setup the tutorials.